### DISCUSSION TITLE
Proper laravel support

### DISCUSSION THREAD

**Discussion by kirkbushell** [freekmurze/laravel-glide#93] [Ideas]
Hi there! First, thanks heaps for the package - useful, but I fear it doesn't go far enough.

Right out the gate I recognised that I needed for it to be more closely involved with the Laravel way of doing things, namely - ability to use the cache filesystem, using storage.etc.

As it stands, as much as I'd like to use it - I just can't, as it doesn't go far enough.

Are there any plans to take it a few steps further?

**Comment by freekmurze** [comment]
There are currently no plans for this. 

You might be interested in using this one https://spatie.be/docs/laravel-medialibrary
